### PR TITLE
Scale freetext font size to closer match web

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-13253
affects: Student
release note: Fixed an issue with text annotations on a submission

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: Core/Core/DocViewer/PSPDFAnnotationExtension.swift [hunk 1]
```diff
@@ -58,12 +58,11 @@ extension PSPDFAnnotation {
             annotation = strikeout
         case .freetext:
             let freeText = PSPDFFreeTextAnnotation(contents: apiAnnotation.contents ?? "")
-            let fontInfo = apiAnnotation.font?.split(separator: " ")
             let fontSizeStr = apiAnnotation.font?.trimmingCharacters(in: CharacterSet(charactersIn: "0123456789").inverted) ?? ""
-            freeText.fontName = fontInfo?.last.flatMap { String($0) } ?? "Verdana"
-            freeText.fontSize = CGFloat(Int(fontSizeStr) ?? 14)
-            freeText.fillColor = .white
-            freeText.sizeToFit()
+            freeText.fontName = "Helvetica" // apiAnnotation.font?.split(separator: " ")?.last.flatMap { String($0) } ?? "Helvetica"
```
Comment (armstrongnate) [inline Core/Core/DocViewer/PSPDFAnnotationExtension.swift]: Did you mean to leave the comment?

File: Core/Core/DocViewer/PSPDFAnnotationExtension.swift [hunk 2]
```diff
@@ -58,12 +58,11 @@ extension PSPDFAnnotation {
             annotation = strikeout
         case .freetext:
             let freeText = PSPDFFreeTextAnnotation(contents: apiAnnotation.contents ?? "")
-            let fontInfo = apiAnnotation.font?.split(separator: " ")
             let fontSizeStr = apiAnnotation.font?.trimmingCharacters(in: CharacterSet(charactersIn: "0123456789").inverted) ?? ""
-            freeText.fontName = fontInfo?.last.flatMap { String($0) } ?? "Verdana"
-            freeText.fontSize = CGFloat(Int(fontSizeStr) ?? 14)
-            freeText.fillColor = .white
-            freeText.sizeToFit()
```
Comment (armstrongnate) [inline Core/Core/DocViewer/PSPDFAnnotationExtension.swift]: I know this was added to fix some really bad text clipping. It's possible PSPDFKit has fixed things though. I do wonder if it's safe to keep around though? Not blocking, want to know what you think.

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1007020&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: Core/Core/DocViewer/PSPDFAnnotationExtension.swift (hunk 1 above)
Comment (vanwagonet) [reply Core/Core/DocViewer/PSPDFAnnotationExtension.swift]: yes. I haven't heard back from DocViewer peeps if always using Helvetica is intentional. This logic isn't anywhere else and I didn't want to lose it.

File: Core/Core/DocViewer/PSPDFAnnotationExtension.swift (hunk 2 above)
Comment (vanwagonet) [reply Core/Core/DocViewer/PSPDFAnnotationExtension.swift]: We set the bounding box further down. Whatever size this determines is thrown away.

**Comment by inst-danger** [comment]
#### Release Note: 
Fixed an issue with text annotations on a submission

#### Affected Apps: Student

[MBL-13253](https://instructure.atlassian.net/browse/MBL-13253)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.23% | 93.24% | -0.01%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**